app/containers/RegisterEmail/actions.js

```diff
@@ -83,6 +83,15 @@ export function resetState() {
   };
 }
 
+function ValidateEmail(mail)   
+{  
+ if (/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(myForm.emailAddr.value))  
+  {  
+    return (true)  
```

Comment:
CMIIW `return true` without the () should work I think, same for the other cases too.